supporting_scripts/hyperion/consistency-check-benchmark/consistency_check.py

```
@@ -170,27 +253,32 @@ def consistency_check_request(session: requests.Session, server_url: str, exerci
     if response.status_code == 200:
         logging.info(f"[{debug_id}] Finished consistency check for programming exercise ID: {debug_id}")
         return response.json()
     else:
-        logging.error(f"Failed to check consistency for programming exercise ID {debug_id}; Status code: {response.status_code}\nResponse content: {response.text}")
+        logging.error(f"Step 12 failed: Consistency check request failed for {debug_id}; Status code: {response.status_code}\nResponse content: {response.text}")
+        logging.error("Execute Step 12 in consistency_check.py")
         return None
 
 if __name__ == "__main__":
+    # This file can be executed independently
+    # NOTE: Steps 11–12 require a session and course id — always use Steps 5–6 and 8 together with them.
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)